tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -121,24 +121,35 @@ else
   "${KOPS_B}" edit cluster "${CLUSTER_NAME}" "--set=cluster.spec.kubernetesVersion=${K8S_VERSION_B}"
 fi
 
-"${KOPS_B}" update cluster
-"${KOPS_B}" update cluster --admin --yes
-# Verify no additional changes
-"${KOPS_B}" update cluster
+if [[ "${KOPS_VERSION_B}" =~ 1.3[123456789] ]]; then
+  # Preview changes
+  "${KOPS_B}" update cluster
```

Comment:
Would be nice if `kops reconcile cluster` would show the same output as `kops update cluster`.

Reply:
We've done this now, so switched to use `reconcile cluster` here